tailwind.config.ts

```diff
@@ -3,10 +3,11 @@ import { withShurikenUI } from './src'
 export default withShurikenUI({
   content: [
     // './.storybook/*.ts',
+    './src/**/*.index.ts',
```

Comment:
this file should not be added as the plugins already register them

Reply:
Removed file from configuration